TinyConstraints/Classes/TinyConstraints+superview.swift

```diff
@@ -164,7 +164,7 @@ public struct LayoutEdge: OptionSet {
     public static let none = LayoutEdge(rawValue: 1 << 6)
 }
 
-public extension TinyView {
+@MainActor public extension TinyView {
```

Comment:
Not needed since `TinyView` is already isolated to the `@MainActor`

```suggestion
public extension Constraint {
```